soc/nordic/common/Kconfig.peripherals

```diff
@@ -207,6 +207,9 @@ config HAS_HW_NRF_RADIO_IEEE802154
 config HAS_HW_NRF_RADIO_TX_PWR_HIGH
 	def_bool $(dt_nodelabel_bool_prop,radio,tx-high-power-supported)
 
+config HAS_HW_NRF_RADIO_CS
```

Comment:
Please keep the alphabetical order here.

Reply:
thank you, I moved it according to alphabetical order